### DISCUSSION TITLE
I can't start custom runner as service

### DISCUSSION THREAD

**Discussion by hannesa2** [community/community#42453] [Actions]
### Custom Runner

Product Feedback

### Body

I'm able to run my arm64 custom runner with `./run.sh` but I want to have it auto started and running as service on my MacOS. 

I followed this guide https://docs.github.com/en/actions/hosting-your-own-runners/configuring-the-self-hosted-runner-application-as-a-service?platform=mac  so I did
```
./svc.sh install                                                               
Creating launch runner in /Users/hannes/Library/LaunchAgents/actions.runner.gitx-gitx.MacStadium.plist
Creating /Users/hannes/Library/Logs/actions.runner.gitx-gitx.MacStadium
Creating /Users/hannes/Library/LaunchAgents/actions.runner.gitx-gitx.MacStadium.plist
Creating runsvc.sh
Creating .service
svc install complete
```
```
./svc.sh status
status actions.runner.gitx-gitx.MacStadium:

/Users/hannes/Library/LaunchAgents/actions.runner.gitx-gitx.MacStadium.plist

Stopped
```
but when I try to run
```
./svc.sh start 
starting actions.runner.gitx-gitx.MacStadium
Failed: failed to load /Users/hannes/Library/LaunchAgents/actions.runner.gitx-gitx.MacStadium.plist
```

How can I solve this ?

**Comment by ADTC** [comment]
Quoting here, for anyone who may need it: https://github.com/actions/runner/issues/1056#issuecomment-1201904324

> It would be great if there were a purely ssh solution that didn't depend on the GUI at all. In the meantime, this can be helpful: go to System Preferences -> Users and Groups -> Login Options -> Automatic Login, and set the 'runner' or 'gha' user to login automatically. Reboot. The user will login at boot time and the services will start.
